.changeset/stale-hairs-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': minor
```

Comment:
is this adding any new functionality? there's no change to the api i don't think?

```suggestion
'@guardian/source-react-components': patch
```

Reply:
Sure!